pxf/build.gradle

```diff
@@ -41,17 +41,22 @@ buildscript {
     }   
 }
 
-// Hadoop distribution (can be phd or hdp) - needed to set correct classpaths
+// Hadoop distribution (can be hdp or null which would default to apache) - needed to set correct classpaths
 // can be set with -Dhd=<value>
-def hddist = System.properties['hd'] ?: 'hdp';
-if (hddist != 'phd' && hddist != 'hdp') {
+def hddist = System.properties['hd'] ?: '';
```

Comment:
As long as we don't have default value, shouldn't it be just: def hddist = System.properties['hd']?